src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -131,6 +178,10 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
     onDescriptionChange = (newDescription: string): void => {
         const { api }: Props = this.props;
         const { file }: State = this.state;
+        if (!file) {
```

Comment:
It doesn't seem like this case can actually occur, since we return null in render if file is falsy.

Reply:
I added it to appease the flow gods